doc/api/fs.md

```diff
@@ -2568,6 +2568,9 @@ changes:
 * `options` {Object}
   * `bigint` {boolean} Whether the numeric values in the returned
     [`fs.Stats`][] object should be `bigint`. **Default:** `false`.
+  * `throwIfNoEntry` {boolean} Whether an exception will be thrown
+    if no file system entry exists, rather than returning `undefined`.
+    **Default:** `true`.
```

Comment:
Could the history be updated to reflect when this was added please?

Reply:
@merceyz Sorry, I'm not familiar with that process.  Is there a changelog or something that needs to be updated?  Can you point me at a sample or a doc?

Reply:
I assume the history a few lines above this is where it's added but I'm also not familiar with it

Reply:
Huh, I didn't see that.  I guess I'm hoping it's generated, because I don't know how I'd know what version to use.

Reply:
It isn't i'm afraid cc @MylesBorins

Reply:
Some of the history items above (e.g. https://github.com/nodejs/node/commit/7b5b8ef42474c7370740c1feefc41330c44ce1ea) make it look like the changelog was updated?

Reply:
It's in the changelog yes but not the history https://nodejs.org/docs/latest-v15.x/api/fs.html#fs_fs_statsync_path_options

Reply:
I see.  So you'd like a new PR with `Added in: v15.3.0` in each of the new sections?

Reply:
Yes please :)